src/Polly.Core.Tests/Registry/ResilienceStrategyRegistryTests.cs

```diff
@@ -55,6 +55,24 @@ public void Clear_Ok()
         registry.TryGet("C", out _).Should().BeTrue();
     }
 
+    [Fact]
+    public void Clear_Generic_Ok()
+    {
+        var registry = new ResilienceStrategyRegistry<string>();
+
+        registry.TryAddBuilder<string>("C", (_, b) => b.AddStrategy(new TestResilienceStrategy()));
+
+        registry.TryAdd("A", new TestResilienceStrategy<string>());
```

Comment:
Do we have asserts somewhere that verify that the Try methods' return value is correct?

Reply:
Yes, there is already test for this. I modified it slightly.